.dockerignore

```diff
@@ -0,0 +1,9 @@
+build_*
+IncludeOS_install/
+diskimagebuild/build
+vmbuild/build
+src/chainload/build
+test/
+examples/
+*/build/
+*/*/build/
```

Comment:
You can do this for directories in dockerignore: `**/build` I think. See: https://docs.docker.com/engine/reference/builder/#dockerignore-file